build/ci/create_demo_pr.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -xe
+set +o pipefail
+
+sed -i "s+$operator_image_in_branch+$operator_image_for_test+g" $csv_file
+
+echo $github_token > github_token.txt
+gh auth login --with-token < github_token.txt
+gh repo fork operator-framework/community-operators --clone community-operators-fork
+cd community-operators-fork
+git remote set-url origin https://csiblock:$github_token@github.com/csiblock/community-operators.git
+git fetch upstream
+git rebase upstream/master
+git push origin master --force
+git config --global user.email csi.block1@il.ibm.com
+git config --global user.name csiblock
```

Comment:
shouldn't this also be in prepare env?

Reply:
I can't pass it to another stage and it will stay for the next stage